cocos/serialization/deserialize-dynamic.ts

```diff
@@ -281,6 +265,28 @@ function compileDeserializeNative (_self: _Deserializer, klass: CCClassConstruct
     };
 }
 
+/**
+ * Tells if the class can be deserialized in "fast mode".
+ * In fast mode, deserialization of the class will go into an optimized way:
+ * each class property will be examined whether to be primitive according to their default value
+ * and type. Finally, all primitive properties would be together deserialized using simple assignment,
+ * without performing in-loop check.
+ */
+function canBeDeserializedInFastMode (klass: any): boolean {
+    return misc.BUILTIN_CLASSID_RE.test(js.getClassId(klass));
+}
+
+function checkIsPrimitiveTypeInFastMode (defaultValue: any, userType: any): boolean {
```

Comment:
As these two functions are check whether can treat them as fast mode, and TypeScript can not override function, so i suggest to rename them to
- canBeDeserializedInFastModeForClass
- canBeDeserializedInFastModeForType

Reply:
> As these two functions are check whether can treat them as fast mode, and TypeScript can not override function, so i suggest to rename them to
> 
> * canBeDeserializedInFastModeForClass
> * canBeDeserializedInFastModeForType

No, the function you outlined is not what you expressed. It expresses "in fast mode, is a property with `defaultValue` and `userType`" is primitive?

Reply:
Then what does fast mode mean for the two functions?

Reply:
fast mode in two functions have different meaning?